_component_design/loading-indicators.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Loading indicators
+parent: Processing
+layout: component
+category: Components
+usage: >
+  Loading indicators tell the user that their system is actively retrieving data.
+preview-image: preview-images/loading.svg
+resource: true
+status: unknown
+last-modified: 2017-10-24
+---
+
+{% include toc.html %}
+
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+{: .hxSubSectionTitle}
+
+Use loading indicators when you need to give the user visual feedback that the system is retrieving the content to be displayed.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-use.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+### Best practice
+{: .hxSubSectionTitle}
+
+- Make sure loading indicators are visible and clearly show the user that information is being retrieved.
+
+- Center loading spinners both vertically and horizontally.
```

Comment:
Perhaps, reword sentence and use singular 'spinner' since most of the doc uses spinner.